tests/src/test/java/alluxio/client/cli/job/DistributedCommandsCancelStatsTest.java

```diff
@@ -43,34 +44,23 @@
  * The tests compare the job statuses (CANCEL or not) and stat counter values for each status.
  */
 public class DistributedCommandsCancelStatsTest extends JobShellTest {
-  static final int TEST_TIMEOUT = 45;
+  private static final String TEST_USER = "test";
+  private static final long SLEEP_MS = Constants.SECOND_MS / 2;
+  private static final int TEST_TIMEOUT = 45;
 
   @ClassRule
-  public static LocalAlluxioClusterResource sResource =
-      new LocalAlluxioClusterResource.Builder()
-          .setNumWorkers(4)
-          .setProperty(PropertyKey.MASTER_PERSISTENCE_CHECKER_INTERVAL_MS, "10ms")
-          .setProperty(PropertyKey.MASTER_PERSISTENCE_SCHEDULER_INTERVAL_MS, "10ms")
-          .setProperty(PropertyKey.JOB_MASTER_WORKER_HEARTBEAT_INTERVAL, "200ms")
-          .setProperty(PropertyKey.WORKER_RAMDISK_SIZE, SIZE_BYTES)
-          .setProperty(PropertyKey.USER_BLOCK_SIZE_BYTES_DEFAULT, SIZE_BYTES)
-          .setProperty(PropertyKey.MASTER_TTL_CHECKER_INTERVAL_MS, Integer.MAX_VALUE)
-          .setProperty(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT, "CACHE_THROUGH")
-          .setProperty(PropertyKey.USER_FILE_RESERVED_BYTES, SIZE_BYTES / 2)
-          .setProperty(PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED, true)
-          .build();
-
-  @BeforeClass
-  public static void beforeClass() throws Exception {
-
-    sLocalAlluxioCluster = sResource.get();
-    sLocalAlluxioJobCluster = new LocalAlluxioJobCluster();
-    sLocalAlluxioJobCluster.start();
-    sFileSystem = sLocalAlluxioCluster.getClient();
-    sJobMaster = sLocalAlluxioJobCluster.getMaster().getJobMaster();
-    sJobShell = new alluxio.cli.job.JobShell(ServerConfiguration.global());
-    sFsShell = new FileSystemShell(ServerConfiguration.global());
-  }
+  public static UnderFileSystemFactoryRegistryRule sUnderfilesystemfactoryregistry =
+        new UnderFileSystemFactoryRegistryRule(new SleepingUnderFileSystemFactory(
+            new SleepingUnderFileSystemOptions()
+                .setIsDirectoryMs(SLEEP_MS)
+                .setIsFileMs(SLEEP_MS)
+                .setGetStatusMs(SLEEP_MS)
+                .setListStatusMs(SLEEP_MS)
+                .setListStatusWithOptionsMs(SLEEP_MS)));
+
+  @Rule
+  public AuthenticatedUserRule mAuthenticatedUser = new AuthenticatedUserRule(TEST_USER,
```

Comment:
why is this necessary?

Reply:
Followed existing tests like LoadMetadataIntegrationTest and ConcurrentDeleteIntegrationTest etc, which all have AuthenticatedUserRule defined.